src/codeu/chat/client/core/Context.java

```diff
@@ -22,15 +22,19 @@
 import codeu.chat.common.User;
 import codeu.chat.util.Uuid;
 import codeu.chat.util.connections.ConnectionSource;
+import codeu.chat.common.ServerInfo;
+import codeu.chat.client.core.View;
 
 public final class Context {
 
   private final BasicView view;
+  private final View view1;
```

Comment:
I changed the variable name and added an explanation for why I added the second view variable